e2e/src/pages/app.po.ts

```diff
@@ -34,6 +34,8 @@ export class AppPage {
         this.clickElement(by.css('label[for="zoom-btn-toggle"]')),
         this.clickElement(by.css('label[for="toggleAnnotations"]'))
       ]);
+    } else {
+      console.log('allready checked');
```

Comment:
remove this please

Reply:
That's been added for debugging purpose and will delete that code